plugins/woocommerce/src/Blocks/BlockTypes/OrderConfirmation/Status.php

```diff
@@ -40,14 +40,24 @@ protected function render( $attributes, $content, $block ) {
 			return '';
 		}
 
-		$additional_content = $this->render_account_notice( $order ) . $this->render_confirmation_notice( $order );
+		$account_notice = $this->render_account_notice( $order );
+
+		if ( $account_notice ) {
+			$block = sprintf(
+				'<div class="wc-block-order-confirmation-status-notices %1$s">%2$s</div>',
+				esc_attr( trim( $classname ) ),
+				$account_notice
```

Comment:
Noting that this is escaped during `wc_print_notice` so it's fine to keep it as-is here.